mysql-test/suite/ndb/r/ndb_basic_ndbd_shm.result

```diff
@@ -915,12 +911,9 @@ id	parent_id
 DROP TABLE child, parent;
 CREATE TABLE t1 (a INT PRIMARY KEY, b TEXT)
 ENGINE=ndb PARTITION BY KEY(a) PARTITIONS 144;
-ERROR HY000: Can't create table 't1' (use SHOW WARNINGS for more info).
 show warnings;
 Level	Code	Message
-Warning	1296	Got error 1224 'Too many fragments' from NDB
-Warning	1296	Failed to commit NDB schema transaction
-Error	1005	Can't create table 't1' (use SHOW WARNINGS for more info).
+drop table t1;
```

Comment:
Even if this change is due to reconfiguration, it seems this test case no longer tests something it used to test. Why allow that test coverage loss?